docs/content/policy-reference.md

```diff
@@ -66,6 +66,13 @@ complex types.
 | <span class="opa-keep-it-together">``output := intersection(set[set])``</span> | ``output`` is the intersection of the sets in the input set  |
 | <span class="opa-keep-it-together">``output := union(set[set])``</span> | ``output`` is the union of the sets in the input set  |
 
+### Objects
+| Built-in | Description |
+| -------- | ----------- |
+| <span class="opa-keep-it-together">``filtered := json.filter(object, paths)``</span> | ``filtered`` is the remaining data from ``object`` with only keys specified in ``paths`` which is an array or set of strings. For example: ``json.filter({"a": {"b": "x", "c": "y"},}, ["a/b"]`` will result in ``{"a": {"b": "x"}}``). |
+
+The ``json`` object paths may reference into array values by using index numbers. For example with the object ``{"a": ["x", "y", "z"]}`` the path `a[1]` references `y`
```

Comment:
Nit: I think this was supposed to be `/a/1` instead of `a[1]` here.

Reply:
Done.